frontend/js/config/merge-profiles/fennica.js

```diff
@@ -92,6 +92,10 @@ import TargetRecord from '../target-record';
 import * as subrecordMergeTypes from '../subrecord-merge-types';
 import { hyphenate } from 'isbn-utils';
 
+var moment = require('moment');
```

Comment:
Import `moment` like this: `import moment from 'moment';`

Also, rebase the branch to merge commits to a single commit

Reply:
Corrected `moment` to be imported correctly. Also commits should be rebased now to single commit.